adapt/tools/text/trie.py

```diff
@@ -157,19 +172,23 @@ def __init__(self, max_edit_distance=0, match_threshold=0.0):
         max_edit_distance and match_threshold.
 
         Args:
-            max_edit_distance(int): ?
-            match_threshold(int): ?
-
-        Notes:
-            This never seems to get called with max_edit_distance or match_threshold
+            max_edit_distance(int): values > 0 allow for fuzzy matching
+                with a maximum levenshtein edit distance
+                https://en.wikipedia.org/wiki/Edit_distance
+            match_threshold(int): only return values with a higher confidence
+                than this value
         """
         self.root = TrieNode('root')
         self.max_edit_distance = max_edit_distance
         self.match_threshold = match_threshold
 
     def gather(self, iterable):
-        """Calls the lookup with gather True Passing iterable and yields
-        the result.
+        """Executes a "gather" traversal of the Trie
+        Result set will include any `is_terminal` nodes encountered during
+        the traversal
+
+        Args
```

Comment:
To keep consistent with the google docstring style (and consistent throughout the file) I think there should be a `:` after Args